proto/process/agent.proto

```diff
@@ -1054,6 +1054,9 @@ message DNSStats {
 	map<uint32, uint32> dnsCountByRcode = 4;
 }
 
+message DNSStatsByQueryType {
+	map<int32, DNSStats> dnsStatsByDomain = 1;
```

Comment:
- There is no domain information at this level. So we probably should change the name from `dnsstatsByDomain` to `dnsStatsByQueryType`.
- What is the mapping from an `int32` to a _QueryType_? We can use an `enum` here like `HTTPMethod`